src/ssl.c

```diff
@@ -30480,6 +30649,30 @@ int wolfSSL_DSA_generate_parameters_ex(WOLFSSL_DSA* dsa, int bits,
     return ret;
 }
 
+WOLFSSL_DSA_SIG* wolfSSL_DSA_SIG_new(void)
```

Comment:
Make sure this only compiles when !NO_DSA.

Reply:
There is an `#ifndef NO_DSA` on line 30480.